docs/docs/end-users/Configuration.fsx

```diff
@@ -0,0 +1,846 @@
+(**
+---
+category: End-users
+categoryindex: 1
+index: 3
+---
+*)
+
+(**
+# Configuration
+Fantomas ships with a limited series of options.
+These can be stored in an [.editorconfig](https://editorconfig.org/) file and will be picked up automatically by the 
+commandline.   
+Your IDE should respect your settings, however the implementation of that is editor specific. Setting the configuration via 
+UI might be available depending on the IDE.
+*)
+
+(**
+## Usage
+Inside .editorconfig you can specify the file extension and code location to be use per config:
+```
+[*.fs]
+fsharp_space_before_uppercase_invocation = true
+
+# Write a comment by starting the line with a '#'
+[*.{fs,fsx,fsi}]
+fsharp_bar_before_discriminated_union_declaration = true
+
+# Apply specific settings for a targeted subfolder
+[src/Elmish/View.fs]
+fsharp_multiline_block_brackets_on_same_column = true
+fsharp_experimental_stroustrup_style = true
+```
+*)
+
+(**
+## Trying your settings via the online tool
+You can quickly try your settings via the <a href="https://fsprojects.github.io/fantomas-tools/#/fantomas/preview" target="_blank">online tool</a>.  
+<img src="{{root}}/online_tool_usage.gif" alt="drawing" width="100%"/>
+*)
+
+#r "nuget: Fantomas.Core, 5.0.0-beta-*"
+
+open Fantomas.Core.FormatConfig
+open Fantomas.Core
+let formatCode input configIndent =
+    CodeFormatter.FormatDocumentAsync  (false, input, configIndent)
+    |>  Async.RunSynchronously
+
+(**
+## Settings recommendations
+Fantomas ships with a series of settings that you can use freely depending  on your case. 
+However, there are settings that we do not recommend and generally should not be used.   
+<p><fantomas-setting-icon type="green"></fantomas-setting-icon><strong>Safe to change:</strong> Settings that aren't attached to any guidelines. Depending on your team or your own preferences, feel free to change these as it's been agreed on the codebase, however, you can always use it's defaults.</p>
+<p><fantomas-setting-icon type="red"></fantomas-setting-icon> <strong>Not recommended:</strong> Settings that don't follow any guidelines.</p>
+<p><fantomas-setting-icon type="orange"></fantomas-setting-icon> <strong>Do not use:</strong> Settings you should never enable. They might lead to incomplete results.</p>
```

Comment:
"Settings you should never enable." is a bit too hard.
Go with "Settings where it is not recommended to change the default value."